### DISCUSSION TITLE
Could not find information on how to setup app proxies

### DISCUSSION THREAD

**Discussion by remchoi** [Shopify/shopify-app-template-node#1207] [Q&A]
Basically, I need to replicate this https://www.youtube.com/watch?v=ZiugtHDctFk with the current version.

**Comment by paulomarg** [comment]
Hi @remchoi! We usually use this repo for reporting bugs on our template, rather than discussions, so your issue might not get a lot of visibility here.

I'd recommend joining the [Shopify Discord channel](https://discord.com/invite/shopifydevs), where the community will be able to help you sort out any specific issues you might be having, and help come up with solutions.
